GliaWidgets/Sources/Screensharing/ScreenShareHandler.Interface.swift

```diff
@@ -0,0 +1,11 @@
+enum ScreenSharingStatus {
+    case started
+    case stopped
+}
+
+struct ScreenShareHandler {
+    var status: () -> ObservableValue<ScreenSharingStatus>
+    var updateState: (CoreSdkClient.VisitorScreenSharingState?) -> Void
+    var stop: ((() -> Void)?) -> Void
```

Comment:
I think readability can be improved by introducing type alias. Something like `Callback`.  So that it could be:
```
var stop: (Callback?) -> Void
```

Reply:
Fixed c17663b6a91b022e075a6c811e0fefa5a9ae7420